src/main/java/io/tackle/operator/KeycloakController.java

```diff
@@ -32,15 +36,33 @@
 
     @Inject
     KubernetesClient kubernetesClient;
+    @Inject
+    PostgreSQLDeployer postgreSQLDeployer;
 
     @Override
     public UpdateControl<Keycloak> createOrUpdateResource(Keycloak keycloak, Context<Keycloak> context) {
         String namespace = keycloak.getMetadata().getNamespace();
         // Keycloak is unique, no need for suffixes in the name
         String name = metadataName(keycloak);
 
+        MixedOperation<Keycloak, KubernetesResourceList<Keycloak>, Resource<Keycloak>> keycloakClient = kubernetesClient.customResources(Keycloak.class);
+        MixedOperation<Tackle, KubernetesResourceList<Tackle>, Resource<Tackle>> tackleClient = kubernetesClient.customResources(Tackle.class);
+        if (tackleClient.inNamespace(namespace).list().getItems().isEmpty()) {
```

Comment:
Shouldn't it be checking for the particular Tackle Operator elements ? Inside the namespace the user could have other deployments, etc, and that would make this fail.